packages/css/src/components/row/row.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/size";
+
+.amsterdam-row {
+  display: flex;
+  flex-direction: row;
+}
+
+@each $size, $label in $amsterdam-sizes {
+  .amsterdam-row--#{$label} {
```

Comment:
Should we include ‘gap’ in this class name: `.amsterdam-row--gap-#{$label}`? If yes, same for `Column`.

Reply:
We're going to offer these classes as utility classes as well, right? If so, shorter is better. OTOH, if we're going to add other props to these components, we have to make a distinction between the rules we're setting. 

Maybe keep it as short as possible for now, and change it when we need to?